src/row.js

```diff
@@ -188,7 +188,7 @@ class Row {
     options = options || {};
 
     families.forEach(function(family) {
-      const familyData = (data[family.name] = {});
+      const familyData = (data[family.id] = {});
```

Comment:
I'd revert this one, for now.  This changes the way rows are read.  The other changes changes aspects of table and family administration.

Once we're ready for this change, this would be a backwards incompatible change for row reads, which is a bit more tricky to convey to users.